services/nublado2/values-idfdev.yaml

```diff
@@ -9,6 +9,7 @@ nublado2:
       hosts: ["data-dev.lsst.cloud"]
       annotations:
         nginx.ingress.kubernetes.io/auth-signin: "https://data-dev.lsst.cloud/login"
+      pathSuffix: "*"
```

Comment:
I think this is no longer needed, right?

Reply:
Oh, right, thanks.